src/data/media-player.ts

```diff
@@ -194,6 +195,20 @@ export const browseLocalMediaPlayer = (
     media_content_id: mediaContentId,
   });
 
+export const getSignedThumbnailPath = async (
+  hass: HomeAssistant,
+  relativeUrl: string
+): Promise<string> => {
+  const thumbnailUrl = new URL(hass.hassUrl(relativeUrl));
```

Comment:
We don't need to make it a `hassUrl` since it will be relative to where we get the frontend from?

Reply:
Or is it required for getSignedPath?

Reply:
The underlying function works only with path: https://github.com/home-assistant/core/blob/dev/homeassistant/components/http/auth.py#L32-L33

Reply:
Ah you also only pass pathname. yeah you can just keep using the relative url.

Reply:
In hindsight, i'd like to delete this. I think i was jaded after having to sign the thumbnail path and had to debug that query parameters were getting in my way, but now i'd rather punt and just use the get signed path function directly from the media browser.